plugins/inputs/postgresql_extensible/README.md

```diff
@@ -44,6 +44,9 @@ The example below has two queries are specified, with the following parameters:
   # Be careful that if the withdbname is set to false you don't have to define
   # the where clause (aka with the dbname)
   #
+  # The script option can be used to speciafy the .sql file path,
```

Comment:
`s/speciafy/specify` then end the sentence on this line, remove `in case` on the next line and just start it with `If script and...`

Same goes for the sampleConfig in the source file (make sure they match)